### PR TITLE
Story/claudia 5942 upload package no ge

### PR DESCRIPTION
**Pull request by hmunfru**
### REVIEWERS

@jesuspg  
### DESCRIPTION

It updates the way to deploy the murano packages when they are attributes. It belongs to teh deployment of no GE packages
### TESTING

Done locally

### CODE REVIEW

**Comment by flopezag** [comment]
The description of the PR should reflect more details that only the person who attends it...

File: package-generator/settings/cookbooks_urls [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-build-essential: https://github.com/chef-cookbooks/build-essential.git
+build-essential: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseRecipes/build-essential/
 apt: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseRecipes/apt-spagobi/
 git: https://github.com/chef-cookbooks/git.git
 yum-mysql-community: https://github.com/chef-cookbooks/yum-mysql-community.git
 yum-epel: https://github.com/chef-cookbooks/yum-epel.git
 apache2: https://github.com/svanzoest-cookbooks/apache2.git
 php: https://github.com/chef-cookbooks/php.git
-mysql: https://github.com/chef-cookbooks/mysql.git
+mysql: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseSoftware/mysql/
```
Comment (flopezag) [inline package-generator/settings/cookbooks_urls:11]: Why do you need to get these products from the forge and not from the chef-cookbooks repository?

Idem for poise...

Comment (hmunfru) [inline package-generator/settings/cookbooks_urls:11]: poise does not work for Centos. We created a PR to that repository. Waiting for it to be accepted. About mysql, we can work with chef repo

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**